app/grandchallenge/components/form_fields.py

```diff
@@ -102,10 +105,27 @@ def get_initial_value(self):
             return self.initial
 
     def get_image_field(self):
+        current_value = None
+
+        if isinstance(self.initial, ComponentInterfaceValue):
+            current_value = self.initial.image
+        elif self.initial:
```

Comment:
I think it makes more sense to move this check up. So first check if `self.initial` is not None and then do the type checks or raise a Runtimeerror.

Reply:
Done